pkg/config/model/types.go

```diff
@@ -117,6 +117,22 @@ type Loader interface {
 	SetEnvKeyReplacer(r *strings.Replacer)
 	SetEnvKeyTransformer(key string, fn func(string) interface{})
 
+	// SetKnown adds a key to the set of known valid config keys
+	SetKnown(key string)
+
+	// API not implemented by viper.Viper and that have proven useful for our config usage
+
+	// BindEnvAndSetDefault sets the default value for a config parameter and adds an env binding
+	// in one call, used for most config options.
+	//
+	// If env is provided, it will override the name of the environment variable used for this
+	// config key
+	BindEnvAndSetDefault(key string, val interface{}, env ...string)
+}
+
+// Compound is an interface for returning compound elements from the config, plus
```

Comment:
Non of the functions of the interface returns any value. I think the comment could be update to reflect that

Reply:
Changed `returning` to `retrieving`